tests/h/routes_test.py

```diff
@@ -132,19 +132,19 @@ def test_includeme():
             request_method="POST",
             factory="h.traversal.BulkAPIRoot",
         ),
-        call("api.groups", "/api/groups", factory="h.traversal.GroupRoot"),
+        call("api.groups", "/api/groups", factory="h.traversal.GroupRequiredRoot"),
```

Comment:
This never uses the subscript look-up, so actually either would do, but a group is certainly not required here.